kanalony-storage-client/src/main/scala/kanalony/storage/generator/EntityClassGenerator.scala

```diff
@@ -0,0 +1,36 @@
+package kanalony.storage.generator
+
+/**
+ * Created by elad.benedict on 2/7/2016.
+ */
+class EntityClassGenerator(val tableMetadata: TableMetadata) {
+
+  private def generateClassColumns(columns: List[IColumnDefinition]) : String = columns match {
```

Comment:
This is a nice use of functional code, but don't you think this statement is more clear:
`columns.map { col => s"${col.name}:${col.typeName}" }.mkString(",\n")`

The last '\n' should be decided by the caller method which is responsible on global text alignment.